src/current/cockroachcloud/connect-to-an-advanced-cluster.md

```diff
@@ -190,7 +192,7 @@ To connect to your cluster from your application:
 
 1. Select the **Connection string** tab.
 1. If the CA certificate for the cluster is not downloaded locally, copy the command to download it. In your terminal, run the command.
-1. Copy the connection string, which begins with `postgresql://`. This will be used to connect your application to CockroachDB {{ site.data.products.advanced }}.
+1. Copy the connection string, which begins with `postgresql://`. This will be used to connect your application to CockroachDB {{ site.data.products.advanced }}. If you are connecting using Private Endpoint in GCP, replace `<ENTER-ENDPOINT-NAME>` in the connection string with the assigned GCP endpoint name.
```

Comment:
```suggestion
1. Copy the connection string, which begins with `postgresql://`. This will be used to connect your application to CockroachDB {{ site.data.products.advanced }}. If you are connecting using private endpoint in GCP, replace `<ENTER-ENDPOINT-NAME>` in the connection string with the assigned GCP endpoint name.
```